cmd/plume/prerelease.go

```diff
@@ -71,7 +85,12 @@ var (
 
 type platform struct {
 	displayName string
-	handler     func(context.Context, *http.Client, *storage.Bucket, *channelSpec, *imageInfo) error
+	handler     func(context.Context, string, *http.Client, *storage.Bucket, *channelSpec, *imageInfo, *map[string]interface{}) error
```

Comment:
Why do both the `platform.handler` & `system.handler` take the `system` string? They should be able to reference the `selectedSystem` variable, similar to what's being done [here](https://github.com/coreos/mantle/pull/934/files#diff-6876a8022c0a64d94d3da2742d1114c3R125) to access the `selectedPlatforms` variable.